seed/serializers/pint.py

```diff
@@ -1,11 +1,109 @@
+"""
+Collects the various utility functions for doing a last-moment collapse of the
+Pint-aware values/columns to raw floats before sending them out over the API.
+Generally this collapsing relies on having access to the organization, since
+that's where the display preference lives.
+"""
+
+import re
 from django.core.serializers.json import DjangoJSONEncoder
 from quantityfield import ureg
 
+AREA_DIMENSIONALITY = '[length] ** 2'
+EUI_DIMENSIONALITY = '[mass] / [time] ** 3'
+
+AREA_DEFAULT_UNITS = 'ft**2'
+EUI_DEFAULT_UNITS = 'kBtu/ft**2/year'
+
+SIGNIFICANT_FIGURES = 2  # this should be... someplace else?
```

Comment:
probably want to make this a user-configurable setting

Reply:
cool, see 756cbefa